### ISSUE TITLE
Kinetics features

### ISSUE THREAD

**Issue by junwenchen** [xumingze0308/TRN.pytorch#14]
Hi,

I am glad to hear that the new result on THUMOS reaches 62.1%.
Can you let me know what TSN weights pretrained by Kinetics are used? 
It would be better if the link to the weights is provided, as there are many Kinetics pretrained weights?

**Comment by xumingze0308** [comment]
We extract visual features using ResNet-50 and motion features using BN-Inception. More details are included in the README.

**Comment by priyamdey** [comment]
Hi. I was experimenting TRN on the TVSeries dataset using the linked pretrained kinetics features. I have preprocessed the raw RGB frames in the following two ways:

1. No resize. Use IN1K mean & std to normalize. Pass through RN50 and avgpool in the end.
2. Resize to 256 and center crop 224. Use either IN1K or TVSeries mean, std to normalize. Send to RN50 & avgpool.

The experiment didn't involve any motion features. Only the camera. And the hidden dim was 2048 (instead of 4096). I got the following result after training & testing using the official code:

1. 71.37%
2. 79.2% (around this mark for both IN1K mean, std and TVSeries mean, std).

Any idea about this discrepancy? Should this happen? Also, which pre-processing was done for paper?

Thanks
